R/req-chunk.R

```diff
@@ -0,0 +1,100 @@
+#' Chunk a request
```

Comment:
Or should this be `Batch a request` and the function name `req_batch()`?

Reply:
Yeah, I like batch better.  `req_` is probably not quite the right prefix (since it doesn't return a request) but I don't see a better alternative.